account_payment_credit_card/models/account.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2017 Open Source Integrators <http://www.opensourceintegrators.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class AccountJournal(models.Model):
+    _inherit = "account.journal"
+
+    credit_card = fields.Boolean('Transfer AP to Credit Card Company')
+    card_processor_partner_id = fields.Many2one(
```

Comment:
@nikul-serpentcs please specify `oldname` during renaming for proper migration

Reply:
@alexey-pelykh Done with oldname.